packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -314,14 +314,6 @@ export const prepareStatusManager = (
         publishEvidence(txHash, evidence);
       },
 
-      /**
-       * Add a new transaction with OBSERVED status
-       * @param {CctpTxEvidence} evidence
-       */
-      observe(evidence) {
-        initPendingTx(evidence, PendingTxStatus.Observed);
-      },
-
```

Comment:
Thanks. I think we also need some cleanup in `status-manager.test.ts`. Some tests can be removed, and others should be updated to use `.advanceSkipped`.